trieste/utils/pareto.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 The Trieste Contributors
```

Comment:
can you import this file into `trieste.utils` please?

Reply:
ok